app/common/directives/ush-logo-react/ush-logo.reducer.js

```diff
@@ -0,0 +1,20 @@
+import { RECEIVE_FORMS } from './ush-logo.reducer';
+
+const initialState = {
+    forms: [
+        { id: 'junk' }
+    ]
+};
+
+export default function ushLogoReducer(state = initialState, action) {
+    switch (action.type) {
+    case RECEIVE_FORMS:
+        return Object.assign({}, state, {
+            forms: action.forms,
+        });
+    default:
+        return state;
+    }
+}
+
+export const getFormsFromState = state => state.forms.forms;
```

Comment:
Kind of a tangent: but I always find it weird that the mappers for a particular reducer have to know what namespace this reducer is used under. ie. if you change the name of `forms` in rootReducer, this code breaks. I don't know if theres a standard solution for that at all, ie only passing the `forms` part of state to this function?